internal/rest/avi_obj_vs.go

```diff
@@ -378,14 +378,25 @@ func (rest *RestOperations) StatusUpdate(rest_op *utils.RestOp, vs_cache_obj *av
 					utils.AviLog.Infof("key: %s, msg: found pool: %s, will update status", key, poolkey.Name)
 					pool_cache_obj, found := pool_cache.(*avicache.AviPoolCache)
 					if found {
-						IPAddr := vs_cache_obj.Vip
-						if vs_cache_obj.Fip != "" {
-							IPAddr = vs_cache_obj.Fip
+						var IPAddrs []string
```

Comment:
This block of code to obtain IPAddrs is being executed multiple times, can we add this to a common function and reuse the code ?